src/component/fetch.js

```diff
@@ -0,0 +1,53 @@
+import React, { useState, useEffect } from 'react';
+
+const FetchData = () => {
+  const [data, setdata] = useState([]);
+  const [load, setload] = useState(false);
+  const [error, seterror] = useState(false);
+
+  useEffect(() => {
+    const category = 'life';
+    const url = `https://api.api-ninjas.com/v1/quotes?category=${category}`;
+    const key = 'JHzAAtqBCo/fC6qrSuPJZw==A8HADyQ1G1Oe8gTe';
+    const fetch = async () => {
+      try {
+        setload(true);
+        const response = await fetch(url, {
+          headers: {
+            'Content-Type': 'application/json',
+            'X-Api-Key': key,
+          },
+        });
+        if (!response.ok) {
+          throw new Error('Failed Request');
+        }
+        const fetchedData = await response.json();
+        setdata(fetchedData);
+        setload(false);
+      } catch (error) {
+        seterror(true);
+        setload(false);
+      }
+    };
+
+    fetch();
```

Comment:
- [ ] Your app gives the `something went wrong` error in the image below because the variable name `fetch` conflicts with the global fetch function in JavaScript. To avoid this conflict, kindly rename the variable to something else, such as `fetchData` for instance.

<img width="820" alt="Screenshot 2023-06-02 at 11 44 57" src="https://github.com/shihabrafio/Math-Magician/assets/64774936/54c500ad-4852-4885-bf66-3839d66eb65b">